qiskit_ibm_runtime/options/estimator_options.py

```diff
@@ -34,86 +34,89 @@
-
-            A configuration is a combination of a specific parameter value binding set and a
-            physical measurement basis. A physical measurement basis groups together some
-            collection of qubit-wise commuting observables for some specific circuit/parameter
-            value set to create a single measurement with basis rotations that is inserted into
-            hardware executions.
-
-            If twirling is enabled, the value of this option will be divided over circuit,
-            randomizations, with a smaller number of shots per randomization. See the
-            :attr:`~twirling` options.
-
-            Default: ``None``.
-
-        optimization_level: (DEPRECATED) How much optimization to perform on the circuits.
-            Higher levels generate more optimized circuits,
-            at the expense of longer processing times.
-
-            * 0: no optimization
-            * 1: light optimization
-
-            Default: 0.
-
-        resilience_level: How much resilience to build against errors.
-            Higher levels generate more accurate results,
-            at the expense of longer processing times.
+    # Sadly we cannot use pydantic's built in validation because it won't work on Unset.
+    default_precision: Union[UnsetType, float] = Unset
+    r"""The default precision to use for any PUB or ``run()``
+        call that does not specify one.
+        Each estimator pub can specify its own precision. If the ``run()`` method
+        is given a precision, then that value is used for all PUBs in the ``run()``
+        call that do not specify their own. 
+        
+        Default: 0.015625 (1 / sqrt(4096)).
+    """
+    default_shots: Union[UnsetType, int, None] = Unset
+    r"""The total number of shots to use per circuit per configuration.
 
-            * 0: No mitigation.
-            * 1: Minimal mitigation costs. Mitigate error associated with readout errors.
-            * 2: Medium mitigation costs. Typically reduces bias in estimators but
-              is not guaranteed to be zero bias.
+        .. note::
+            If set, this value overrides :attr:`~default_precision`.
 
-            Refer to the
-            `Configure error mitigation for Qiskit Runtime
-            <https://docs.quantum.ibm.com/guides/configure-error-mitigation>`_.
-            for more information about the error mitigation methods used at each level.
+        A configuration is a combination of a specific parameter value binding set and a
+        physical measurement basis. A physical measurement basis groups together some
+        collection of qubit-wise commuting observables for some specific circuit/parameter
+        value set to create a single measurement with basis rotations that is inserted into
+        hardware executions.
 
-            Default: 1.
+        If twirling is enabled, the value of this option will be divided over circuit,
```

Comment:
"divided over circuit, randomizations," reads oddly to me but maybe it is codespeak?